lips/lip-6.mdx

```diff
@@ -10,26 +10,30 @@ created: 07/30/2020
-* *Reconfiguration* is an on-chain configuration change. Reconfigurations are separated by epochs signified by a `LibraConfig::NewEpochEvent`.  If a transaction causes a Reconfiguration, that transaction is the last transaction before the next epoch. Any subsequent transactions will be executed in the new configuration defined by the Reconfiguration.
-* *LibraRoot* is the sole entity capable of adding or removing Validator Owner and Validator Operator accounts and specifying the current Validator Owners in the  Validator Set. It may also submit an AdminScript.
+* *Network addresses* define endpoints and cryptographic credentials for establishing secure connections with other Validator Nodes as well as Full Nodes and clients.
+* *Validator Owner* is the entity responsible for the operation of a Validator Node and is a member of the Libra Association. Each Validator Owner has an account on the Libra Blockchain that stores the Validator Config Move resource (see [Move Tutorial](https://developers.libra.org/docs/move-paper)).
+* *Validator Operator* is the entity, approved by LN, that operates a Validator Node. A Validator Operator could be run by a Validator Owner, or a Validator Operator could be a separate entity acting as an operator on behalf of one or more Validator Owners. A Validator Operator may run multiple Validator Nodes. The Validator Operator for a Validator Owner has permission to modify the Validator Config stored in the Validator Owner account and update its copy in the Validator Set.
+* *Validator Set* is an on-chain Move resource represented by a vector containing the active set of Validator Owners. There exists a one-to-one relationship between a Validator Owner in the Validator Set and a Validator Node. The entries within the Validator Set consist of the Validator Owner's address as well as a copy of its Validator Config. Each Validator Owner address must appear only once in the Validator Set. Validator Nodes run according to the configuration defined in the Validator Set. Validator Set changes come in two ways: a change in the set of Validator Owners or a change in the configuration of some Validator Owners. Both cases result in a Reconfiguration.
+* *Reconfiguration* is an on-chain configuration change. Reconfigurations are separated by epochs signified by a LibraConfig::NewEpochEvent. If a transaction causes a Reconfiguration, that transaction is the last transaction within the current epoch. The next epoch starts for subsequent transactions, which will be executed in the new configuration defined by the Reconfiguration.
```

Comment:
The next epoch starts for subsequent transactions => 
Subsequent transactions will be executed in the new epoch with the configuration change that triggered the Reconfiguration.